modules/deploy/pages/deployment-option/cloud/api/cloud-api-overview.adoc

```diff
@@ -0,0 +1,61 @@
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management and manage Redpanda Cloud resources.
+
+== Requirements
+
+To be able to use the Cloud API:
+
+* You must be a customer with an existing organization in Redpanda Cloud.
+* You can only use one organization for authentication. See xref:deploy:deployment-option/cloud/api/cloud-api-authentication.adoc[Cloud API Authentication] for steps to authenticate API requests.
+
+== Cloud API architecture
+
+Redpanda Cloud uses a control plane and data plane architecture. The *control plane* is where most cluster management, operations, and maintenance takes place. The control plane enforces rules in the data plane. The *data plane* is where your cloud cluster lives. The term _data plane_ is used interchangeably with _cluster_. To learn more about the Redpanda Cloud platform, see the xref:deploy:deployment-option/cloud/cloud-overview.adoc[overview].
+
+The *Control Plane API* allows you to interact with the control plane. You use the Control Plane API to manage cloud clusters, organization, resource groups, networks, and other cloud resources. The Control Plane API does not interact directly with Redpanda clusters, although some of the actions performed through the API can eventually make their way to a cluster. See xref:deploy:deployment-option/cloud/api/cloud-controlplane-api.adoc[Manage Clusters using the Cloud API] for steps to create and delete clusters, or see the full xref:api:ROOT:cloud-api.adoc[Cloud API Reference documentation]. 
+
+The *Data Plane APIs* are a collection of APIs through which you can perform actions directly against a Redpanda cluster. These APIs expose more high-level operations on the data plane by acting as a wrapper around the other APIs supported by Redpanda (Kafka API, Admin API, or Schema Registry API). See xref:deploy:deployment-option/cloud/api/cloud-dataplane-api.adoc[Manage Cluster Resources Using the Cloud API] for steps to create an access-control list (ACL), topic, or connector, or see the full xref:api:ROOT:cloud-api.adoc[Cloud API Reference documentation].
+
+NOTE: For more information on the APIs available with Redpanda, see the xref:reference:api-reference.adoc[API and SDK Reference documentation].
+
+== Base URLs
+
+The Control Plane and Data Plane APIs are independent of each other. The Cloud API URL is distinct from the Data Plane API URLs.
+
+=== Control Plane API URL
+
+The Control Plane API has a base URL that is the same for any organization:
+
+----
+https://api.redpanda.com
+----
+
+When making requests to the Control Plane API endpoints, the request URL is the base URL, plus the API version and the resource path, for example:
+
+[,bash]
+----
+https://api.redpanda.com/v1beta2/clusters
+----
+
+The Control Plane API version *v1beta2* is currently supported.
```

Comment:
```suggestion
The current Control Plane API version is *v1beta2*.
```
Not sure if we need to document this here. If so, you'll need to manually update each time it changes.

Reply:
@micheleRP maybe we could use something like attributes to manage these values? @JakeSCahill ?